Improved Klee-stats with basic config file

suryanash opened the pull request:
I have been working on improving klee-stats and adding more functionality to it while also making it more flexible. Klee-stats now can partially parse the run.istats file with the assembly.ll file. The new functionality also comes from outsourcing information to a config file. These config files are dependent on the version of stats klee is outputting. The folder stat-configs is where these config files will be, currently there is one for the first version of the stats, version-1. Klee-stats will also try to automatically select the correct version (which it will find out by reading the first line of the run.stats file) by searching for the corresponding file with the name "version-{v}.config" where {v} is the current version number.
The new features, along with how to use config files have been documented in the README.md files. The one pertaining to the config files is inside the folder stat-configs.

ccadar commented:
That's great work, thanks, Surya!  One thing that would be useful would be a couple of regression tests (which I would put in a separate directory, say tests/klee-stats).  They would also be valuable as documentation.

suryanash commented:
Thanks Cristian. I have now added test cases for the different features and amended the appropriate TravisCI configuration files to run the tests correctly. I have also added the top locations feature which displays the line number for the LLVM instruction that took the most time (along with the mnemonic).

ccadar commented:
See #534
